bears/general/FilenameBear.py

```diff
@@ -71,15 +81,32 @@ def run(self, filename, file,
                      file_naming_convention)
             return
 
+        if new_name != filename_without_extension:
+            messages.append(
+                'Filename does not follow {} naming-convention.'.format(
+                    file_naming_convention))
+
+        if not filename_without_extension.startswith(filename_prefix):
+            new_name = filename_prefix + new_name
+            messages.append(
+                'Filename does not use the {!r} prefix.'.format(
```

Comment:
maybe rephrase slightly to
```
Filename does not use the prefix {!r}
```
Currently it sounds a bit "this is one well-known prefix".